clairvoyance/client.py

```diff
@@ -63,15 +62,19 @@ async def post(
                     log().warning(f"Received status code {response.status}")
                     return await self.post(document, retries + 1)
 
-                return await response.json(content_type=None)
+                try:
+                    return await response.json(content_type=None)
+                except json.decoder.JSONDecodeError as e:
+                    log().warning(f"JSON decode error while decoding response from {self._url} (status code: {response.status}): {e}")
+                    log().debug("[Hint] Endpoint might require authentication, or, site is behind something like Cloudflare and is rate limiting you. You can pass headers and cookies via -H option.")
```

Comment:
It's a significant improvement over previous version. Could we mention troubleshooting guide? I feel that it contains more detailed instructions that might be really helpful.